src/main/kotlin/com/healthmetrix/labres/order/ExternalOrderNumberController.kt

```diff
@@ -100,26 +107,26 @@ class OrderController(
             )
         ]
     )
-    fun getOrderNumber(
+    fun queryStatus(
         @Parameter(
             description = "UUID of an order that has been sent to a lab",
             required = true,
             schema = Schema(type = "string", format = "uuid", description = "A Version 4 UUID")
         )
         @PathVariable orderId: String
     ): ResponseEntity<StatusResponse> {
-        val id = try {
+        val orderId = try {
```

Comment:
I liked the mental model of shadowing it after "deserializing" it so that you never use it in the wrong format. But I aslo learnt that it's rather a bad practice in kotlin, so I will rename it again.